qa/tasks/util/__init__.py

```diff
@@ -234,7 +234,7 @@ def remote_exec(remote, cmd_str, logger, log_spec, quiet=True, rerun=False, trie
             except CommandFailedError:
                 logger.error(("{} failed. Creating /var/log/journalctl.log with "
                               "output of \"journalctl --all\"!").format(log_spec))
-                remote.sh("sudo journalctl --all > /var/log/journalctl.log")
+                remote.sh("sudo journalctl --all | sudo tee /var/log/journalctl.log > /dev/null")
```

Comment:
Good catch, @shyukri - just one suggestion. Instead of piping into "sudo tee" and suppressing the output on stdout, could we do:

```
sudo su -c "journalctl --all > /var/log/journalctl.log"
```